src/coffee/io/writer.coffee

```diff
@@ -0,0 +1,105 @@
+_ = require 'underscore'
+Csv = require 'csv'
+path = require 'path'
+Promise = require 'bluebird'
+iconv = require 'iconv-lite'
+fs = Promise.promisifyAll require('fs')
+Excel = require 'exceljs'
+
+DEBUG = false
+debugLog = if DEBUG then console.log else _.noop
```

Comment:
Agree.. or we can rewrite it directly into ES6 so we will loose dependency on coffeescript which is already dead.